packages/prisma/schema.prisma

```diff
@@ -683,7 +683,7 @@ model Schedule {
   eventType            EventType[]
   instantMeetingEvents EventType[]    @relation("InstantMeetingSchedule")
   name                 String
-  timeZone             String?
+  timeZone             String?        @default("Europe/London")
```

Comment:
Why are we having "Europe/London" as default timeZone 🤔

Reply:
"Europe/London" timezone aligns with UTC or has UTC +0, probably that's why its chosen as default timezone even for users

Reply:
Can we directly use UTC timeZone, I think that would be better 🤔 .

Reply:
to be in sync with default value for timezone in other tables - `User`, `Team` and other places in code, i think it's better to go with `Europe/London`.

Reply:
cool 🆒